Users can now use FQDN names for their hosts (#3393)
elastic/security-docs: docs/release-notes/8.8.asciidoc

[[release-notes-8.8.0]]
=== 8.8.0

> To view a detailed summary of new features and enhancements we've documented this release, check out our {security-guide}/whats-new.html[{minor-version} release highlights].

Reviewer: This is a tricky statement. Changing the link text to an attribute doesn't quite solve the problem because it'll still be a disconnect between the sentence's first part always saying/implying 8.8 (because it's hardwritten into the file) and the second part dynamically saying whatever the latest version is.
```suggestion
To view a detailed summary of the latest features and enhancements, check out our {security-guide}/whats-new.html[release highlights].
```

* Session view: Makes the row representing the session leader remain visible when you scroll past it, and adds a button to this row that allows you to collapse child processes ({pull}154982[#154982]).
* Reduces Linux process event volume by about 50% by combining `fork`, `exec`, and `end` events when they occur around the same time (does not affect queries of this data) ({pull}153213[#153213]).
* Updates where the technical preview tags appear for host risk score features ({pull}156659[#156659], {pull}156514[#156514]).

> * You can now use fully qualified domain names (FQDNs) for hosts. To learn how to enable the FQDN feature flag, refer to {fleet-guide}/elastic-agent-standalone-feature-flags.html[Configure feature flags for standalone {agents}]. To learn how to set host names in {fleet}, refer to {fleet-guide}/fleet-settings.html#fleet-agent-hostname-format-settings[Agent Binary Download {fleet} settings]. 

Reviewer 2: Revising for formatting consistency. 
```suggestion
* Allows you to use fully qualified domain names (FQDNs) for hosts. To learn how to enable the FQDN feature flag, refer to {fleet-guide}/elastic-agent-standalone-feature-flags.html[Configure feature flags for standalone {agents}]. To learn how to set host names in {fleet}, refer to {fleet-guide}/fleet-settings.html#fleet-agent-hostname-format-settings[Agent Binary Download {fleet} settings]. 
```

> * You can now use fully qualified domain names (FQDN) for hosts. To learn how to enable the FQDN feature flag, refer to {fleet-guide}/elastic-agent-standalone-feature-flags.html[Configure feature flags for standalone Elastic Agents]. To learn how to set host names in Fleet, refer to {fleet-guide}/fleet-settings.html#fleet-agent-hostname-format-settings[Agent Binary Download Fleet settings]. 

Reviewer: Also I think `Elastic Agents` and `Fleet` should be attributes
```suggestion
* You can now use fully qualified domain names (FQDNs) for hosts. To learn how to enable the FQDN feature flag, refer to {fleet-guide}/elastic-agent-standalone-feature-flags.html[Configure feature flags for standalone Elastic Agents]. To learn how to set host names in Fleet, refer to {fleet-guide}/fleet-settings.html#fleet-agent-hostname-format-settings[Agent Binary Download Fleet settings]. 
```

Author: yeah, you're right - made the updates below

```suggestion
* You can now use fully qualified domain names (FQDNs) for hosts. To learn how to enable the FQDN feature flag, refer to {fleet-guide}/elastic-agent-standalone-feature-flags.html[Configure feature flags for standalone {agent}]. To learn how to set host names in {fleet}, refer to {fleet-guide}/fleet-settings.html#fleet-agent-hostname-format-settings[Agent Binary Download {fleet} settings]. 
```
